spec/factories/products.rb

```diff
@@ -3,5 +3,6 @@
     name { "MyString" }
     description { "MyText" }
     price { 1 }
+    category_id { 1 }
```

Comment:
You can never be sure that you will have a category with id 1. Not even in test env. 

I would just say `category` on that line and let FactoryBot do its job. If you are unsure, try it and research it.